Vagrantfile

```diff
@@ -32,11 +32,11 @@ commandsToCheck = [
   end
 
 require 'yaml'
-AWS = YAML.load_file 'aws.yml'
+CONFIGURATION = YAML.load_file 'configuration.yml'
 
 if File.exist?('aws.yml.local')
     private_settings = YAML::load_file('aws.yml.local')
-    AWS.merge!(private_settings)
+    CONFIGURATION.merge!(private_settings)
 end
```

Comment:
too many configuration files. aws.yml, aws,yml.local, configuration.yml

Reply:
I renamed aws.yml to  configuration.yml
do you prefer to change aws.yml.local to configuration.yml.local